### ISSUE TITLE
citations with newlines fail

### ISSUE THREAD

**Issue by gidden** [elliottbiondo/zo#2]
```
$ ../zo/zo.py make -p bibliography.bib 
Traceback (most recent call last):
  File "../zo/zo.py", line 100, in <module>
    main()
  File "../zo/zo.py", line 97, in main
    make(parent)
  File "../zo/zo.py", line 18, in make
    while line[idx] != "}":
IndexError: string index out of range
```

**Comment by elliottbiondo** [comment]
fixed in 3f622de51985efb54e

**Comment by gidden** [comment]
works like a charm =)

On Wed, Feb 18, 2015 at 11:17 AM, Elliott Biondo notifications@github.com
wrote:

> fixed in 3f622de
> https://github.com/elliottbiondo/zo/commit/3f622de51985efb54ebd4134753c2839e5b514c4
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/elliottbiondo/zo/issues/2#issuecomment-74905115.

## 

Matthew Gidden
Ph.D. Candidate, Nuclear Engineering
The University of Wisconsin -- Madison
Ph. 225.892.3192
